rts/lra/lra-service-base/src/main/java/io/narayana/lra/GenericLRAException.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
```

Comment:
This exception classes are taken from the specification which is under ASL 2.0 but Narayana is licensed under LGPL 2.1. I am not sure if this is a problem I just wanted to highlight it.

Also copyright should probably have year 2019?

Reply:
I removed the eclipse exception classes and mappers.
Thanks for the review.